lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -126,6 +126,7 @@
 
 import json
 import traceback
+from typing import Tuple
```

Comment:
This import isn't needed to avoid runtime errors, since the only use of the type hint is in a comment.

Also, this import won't work on Python 2.x.

Reply:
__pip2 install typing__

Reply:
That would add an additional dependency for the module which provides no benefit at run time.

Reply:
___Typing intentionally does not happen at runtime.___ From [PEP484]{https://www.python.org/dev/peps/pep-0484/#abstract}...
> While these annotations are available at runtime through the usual __annotations__ attribute, no type checking happens at runtime. Instead, the proposal assumes the existence of a separate off-line type checker which users can run over their source code voluntarily.  Essentially, such a type checker acts as a very powerful linter.

Reply:
Correct. However, this still adds an unnecessary dependency to the module.

Reply:
If you do not want to support type hints then do not put them in the code.  If they are in the code then they need to be supported with proper imports.  Can't have it both ways.

Reply:
https://github.com/PyCQA/pylint/issues/647 Still open since 2015